pkgs/development/python-modules/flask-autoindex/default.nix

```diff
@@ -1,28 +1,38 @@
 { lib
 , buildPythonPackage
-, pythonOlder
 , fetchPypi
 , flask
 , flask-silk
 , future
-, pathlib
+, pythonOlder
+, unittestCheckHook
 }:
 
 buildPythonPackage rec {
-  pname = "Flask-AutoIndex";
+  pname = "flask-autoindex";
```

Comment:
I think it would be better to stick to the PyPI name since that is probably what people looking for the package would expect. Do we have a policy to normalize the names of python modules?

Reply:
> Do we have a policy to normalize the names of python modules?

We do, from https://nixos.org/manual/nixpkgs/stable/#contributing:

> Attribute names in python-packages.nix as well as pnames should match the library’s name on PyPI, but be normalized according to [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names). This means that characters should be converted to lowercase and . and _ should be replaced by a single -...

Reply:
Okay, thanks for the reference. In that case it all looks good to me :)